QMigrate/signal/pick/gaussianpicker.py

```diff
@@ -61,18 +56,26 @@ def __init__(self, onset=None, **kwargs):
 
         self.onset = onset
         self.pick_threshold = kwargs.get("pick_threshold", 1.0)
-        self.fraction_tt = kwargs.get("fraction_tt", 0.1)
         self.marginal_window = kwargs.get("marginal_window", 1.0)
         self.sampling_rate = None
         self.plot_picks = kwargs.get("plot_picks", False)
 
+        if "fraction_tt" in kwargs.keys():
+            print("FutureWarning: Fraction of traveltime argument moved to "
+                  "lookup table.\n")
+        self._fraction_tt = kwargs.get("fraction_tt")
```

Comment:
Hmm. This is true, but also still here to support cases where the user wishes to override the LUT fraction_tt to fine-tune the picker. So I think the message should perhaps be revised slightly to include this case.

Reply:
Yep - depends how we want to expose that - the way I've implemented it was for the intention of this being a usable but 'private' interface with the `fraction_tt`.

Reply:
But someone using this for that purpose would still get this error message, right? Not sure what's best in terms of message if that's the case, as it's both a deprecation and a semi-protected variable.

Reply:
Yep - I think we should state somewhere in our deprecation warnings exactly when those deprecations will take effect (v1.1 imo) and set a reasonable timeline v1.1. Until then, maybe we could word the message better...

` - it remains possible to override the fraction of traveltime here, if required, to further tune the phase picker.`